@navikt/core/react/src/date/monthpicker/MonthCaption.tsx

```diff
@@ -0,0 +1,109 @@
+import { Left, Right } from "@navikt/ds-icons";
+import { isSameYear, setYear, startOfMonth, startOfYear } from "date-fns";
+import React from "react";
+import { useDayPicker } from "react-day-picker";
+import { Button, Select } from "../..";
+import {
+  hasNextYear,
+  updateWithoutDropdownCaption,
+  updateWithDropdownCaption,
+} from "../utils/handle-selected";
+import { labelPrevYear, labelNextYear } from "../utils";
+
+export const MonthCaption = ({
+  selected,
+  onSelect,
+  isValidDropdownCaption,
+  dropdownCaption,
+  yearState,
+  setYearState,
+}: {
+  selected: Date;
+  onSelect: (m: Date) => void;
+  isValidDropdownCaption: boolean;
+  dropdownCaption: boolean;
+  yearState: Date;
+  setYearState: Function;
+}) => {
+  const {
+    fromDate,
+    toDate,
+    formatters: { formatYearCaption },
+    locale,
+  } = useDayPicker();
+
+  const years: Date[] = [];
+
+  if (dropdownCaption && fromDate && toDate) {
+    const fromYear = fromDate.getFullYear();
+    const toYear = toDate.getFullYear();
+    for (let year = fromYear; year <= toYear; year++) {
+      years.push(setYear(startOfYear(new Date()), year));
+    }
+  }
+
+  const handleYearChange = (e) => {
+    const newMonth = setYear(startOfMonth(selected), Number(e.target.value));
+    setYearState(newMonth);
+  };
+
+  const handleButtonClick = (val) => {
+    let newMonth: Date;
+    if (isValidDropdownCaption && hasNextYear(yearState, years, val)) {
+      newMonth = updateWithDropdownCaption(yearState, selected, years, val);
+      setYearState(newMonth);
+    } else if (!isValidDropdownCaption) {
+      newMonth = updateWithoutDropdownCaption(yearState, val);
+      setYearState(newMonth);
+    }
+  };
+
+  const hasFollowingYear = (value: number) => {
+    return years.some((y) =>
+      isSameYear(y, setYear(yearState, Number(yearState.getFullYear() + value)))
+    );
+  };
+
+  return (
+    <div className="navds-date__caption">
+      <Button
+        className="navds-date__caption-button navds-button"
+        disabled={!isValidDropdownCaption ? false : !hasFollowingYear(-1)}
+        onClick={() => handleButtonClick(-1)}
+        aria-label={labelPrevYear(locale?.code)}
+        icon={<Left aria-hidden />}
+        variant={"tertiary"}
```

Comment:
```suggestion
        variant="tertiary"
```

Finnes det en eslint-regel vi som plukker opp slik, om så har det vært nice å fått inn i package.json

Reply:
Ser ut til at vi kan bruke `react/jsx-curly-brace-presence`